content/posts/Ayan_Phase_1_Report.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "LMS Integration: Phase 1 Report (GSoC 2021- Ayan Biswas)"
+date: 2021-07-02T21:32:37+05:30
+draft: false
+author: Ayan Biswas
+type: post
+---
+
+![GSoC](/images/gsoc_cover.png)
+
+This blog describes my works done for 📚 [LMS Integration](https://summerofcode.withgoogle.com/projects/#6282924346834944) project during Google Summer of Code 2021 🖥.
+
+The overall goal of the project is to make CircuitVerse compatible with as much as LMSs possible.
+
+#### About my Project
+
+Currently CircuitVerse supports it's inbuilt LMS(Learing Management System) where teacher can directly create assignments and students can submit circuits in the form of projects at CircuitVerse, and after evaluations teacher grades them.
+
+Lot of institutions uses their own lms (like Moodle,Canvas etc) for their academic works and which are obviously not compatible with our platform CircuitVerse, our main challenge is to integrate as much as many LMSs with CircuitVerse such that a teacher can use CircuitVerse without leaving their favourite LMSs.
+
+#### Major Goals Achieved in First Phase
+
+Here are the changes that I have done before the first evaluation
+
+* The codebase of CircuitVerse made [LTI (Learning Tools Interoperability)](https://www.imsglobal.org/activity/learning-tools-interoperability) compatible such that any LMS (which supports LTI) can be connected with the assignments of CircuitVerse.
+
+* The **UI developed for generation of the required keys and secrets** to connect an assignment with an external LTI supported LMS (like Moodle/Canvas).
```

Comment:
Rephrase: **UI was developed for generation of the required keys and secrets** to connect an assignment with an external LTI supported LMS (like Moodle/Canvas).